docs/tutorial/package.md

```diff
@@ -97,12 +92,11 @@ You can see that you have a generated project structure that looks like:
 .
 ├── poetry.lock
 ├── pyproject.toml
-├── README.rst
+├── README.md
```

Comment:
@kinuax, thanks for this, it looks like you've carefully went through the docs to ensure consistency, which is super appreciated 🙏 

I haven't reviewed this PR in full detail yet, but I wanted to note that the `rst` -> `md` fix is already addressed by https://github.com/tiangolo/typer/pull/632, so I would recommend (to Tiangolo) to merge that one first, then update this one, to keep the number of changes per PR minimal.

Reply:
PR is updated to consider already merged fix and to avoid conflicts.